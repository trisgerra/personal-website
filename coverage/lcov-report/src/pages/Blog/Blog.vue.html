<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/pages/Blog/Blog.vue</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/pages/Blog</a> Blog.vue
    </h1>
    <div class='clearfix'>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">&lt;template&gt;
  &lt;div&gt;
    &lt;h1&gt;Blog&lt;/h1&gt;
    &lt;div class="masonry"&gt;
        &lt;div class="article"
        v-bind:key="article.title"
         v-for="article in articles"&gt;
            &lt;h1&gt; {{ article.title }}&lt;/h1&gt;
        &lt;/div&gt;
    &lt;/div&gt;
  &lt;/div&gt;
&lt;/template&gt;
&nbsp;
&lt;script&gt;
import Prismic from '@prismicio/client';
&nbsp;
export default {
  data() {
<span class="cstat-no" title="statement not covered" >    return {</span>
      articles: [],
    };
  },
  methods: {
    getContent() {
<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >      this.$prismic.client.query(</span></span>
        Prismic.Predicates.at('document.type', 'test'),
      ).then((document) =&gt; {
<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >        this.articles = document.results.map(item =&gt; item.data);</span></span>
<span class="cstat-no" title="statement not covered" >        debugger;</span>
      });
    },
  },
&nbsp;
  created() {
<span class="cstat-no" title="statement not covered" >    this.getContent();</span>
  },
};
&lt;/script&gt;
&nbsp;
&lt;style lang="scss"&gt;
.masonry {
    display: grid;
    grid-gap: 10px;
    grid-template-columns: repeat(auto-fill, minmax(250px,1fr));
    grid-auto-rows: 20px;
}
&lt;/style&gt;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Apr 23 2021 01:35:58 GMT+0200 (Central European Summer Time)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
