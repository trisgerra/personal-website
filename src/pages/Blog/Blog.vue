<template>
  <div class="blog">
    Blog
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component({
  components: {
  },
})
export default class Blog extends Vue {}
</script>
