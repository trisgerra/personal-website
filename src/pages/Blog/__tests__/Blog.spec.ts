import { Wrapper } from '@vue/test-utils';
import Blog from '../Blog.vue';
import createWrapper from '@/utils/CreateWrapper';

describe('Design', () => {
  it('should render as expected', () => {
    const wrapper: Wrapper<Blog> = createWrapper(Blog);
    expect(wrapper.element).toMatchSnapshot();
  });
});
