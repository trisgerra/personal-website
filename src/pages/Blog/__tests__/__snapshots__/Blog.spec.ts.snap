// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Design should render as expected 1`] = `
<div
  class="blog"
>
  
  Blog

</div>
`;
